src/main/resources/init_db.sql

```diff
@@ -5,3 +5,43 @@ CREATE TABLE `internet_taxi`.`manufacturers` (
   `manufacturer_country` VARCHAR(225) NOT NULL,
   `manufacturer_deleted` VARCHAR(225) NOT NULL DEFAULT 'FALSE',
   PRIMARY KEY (`manufacturer_id`));
+
+CREATE TABLE `internet_taxi`.`cars` (
+  `car_id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+  `manufacturer_id` BIGINT(11) NOT NULL,
+  `car_model` VARCHAR(225) NOT NULL,
+  `deleted` BIT(1) NOT NULL DEFAULT false,
+  PRIMARY KEY (`car_id`));
+
+ALTER TABLE `internet_taxi`.`cars`
+ADD INDEX `cars_manufacturers_fk_idx` (`manufacturer_id` ASC) VISIBLE;
+;
+ALTER TABLE `internet_taxi`.`cars`
+ADD CONSTRAINT `cars_manufacturers_fk`
+  FOREIGN KEY (`manufacturer_id`)
+  REFERENCES `internet_taxi`.`manufacturers` (`manufacturer_id`)
+  ON DELETE NO ACTION
+  ON UPDATE NO ACTION;
+
+CREATE TABLE `internet_taxi`.`drivers` (
+  `driver_id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+  `driver_name` VARCHAR(225) NOT NULL,
+  `driver_license_number` VARCHAR(225) NOT NULL,
+  `deleted` BIT(1) NOT NULL DEFAULT false,
+PRIMARY KEY (`driver_id`));
```

Comment:
```suggestion
CREATE TABLE `internet_taxi`.`drivers` (
  `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
  `name` VARCHAR(225) NOT NULL,
  `license_number` VARCHAR(225) NOT NULL,
  `deleted` BIT(1) NOT NULL DEFAULT false,
PRIMARY KEY (`driver_id`));
```
there is no need to indicate `driver_` prefix for each column because the columns are located in `drivers` table

same for other column names in `manufacturer` and `car` tables